Command/CleanMediaCommand.php

```diff
@@ -25,11 +25,6 @@
 class CleanMediaCommand extends ContainerAwareCommand
 {
     /**
-     * @var MediaProviderInterface[]|false
-     */
-    private $providers = false;
```

Comment:
> Clean media command was not working because it was trying to access to a providers property that does not exists.

I don't get it… looks like the property does exist, doesn't it?